python/cuml/test/dask/test_random_forest.py

```diff
@@ -334,14 +335,67 @@ def test_rf_regression_dask_cpu(partitions_per_worker, cluster):
         X_train_df, y_train_df = dask_utils.persist_across_workers(
             c, [X_train_df, y_train_df], workers=workers)
 
-        cu_rf_mg = cuRFR_mg(**cu_rf_params)
-        cu_rf_mg.fit(X_train_df, y_train_df)
+        cuml_mod = cuRFR_mg(**cu_rf_params)
+        cuml_mod.fit(X_train_df, y_train_df)
 
-        cu_rf_mg_predict = cu_rf_mg.predict(X_test, predict_model='CPU')
+        cuml_mod_predict = cuml_mod.predict(X_test, predict_model='CPU')
 
-        acc_score = r2_score(cu_rf_mg_predict, y_test)
+        acc_score = r2_score(cuml_mod_predict, y_test)
 
         assert acc_score >= 0.67
 
     finally:
         c.close()
+
+
+@pytest.mark.parametrize('partitions_per_worker', [1])
+@pytest.mark.parametrize('output_class', [True, False])
+def test_rf_classification_dask_fil_predict_proba(partitions_per_worker,
+                                                  cluster,
+                                                  output_class):
+
+    # Use CUDA_VISIBLE_DEVICES to control the number of workers
```

Comment:
Not sure I understand this comment -- looks like you've specified n_workers=1 below which should fix n_workers even with multiple devices?

Reply:
my bad, this is a comment from the above test function on which the predict proba test is based. I will remove it.